packages/terra-heading/tests/nightwatch/heading-spec.js

```diff
@@ -0,0 +1,58 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
I would tend to agree the API should generally be tests for components, however,  these components (text and heading) are being created specifically for the css styling. This makes me think it is important to test the css implementation. Others?